client_admin/test/unit/test_admin_status.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+from mock import Mock
```

Comment:
In Python 2, mock is not part of the standard library, so for now we should put it in the third party import group.